scripts/deployment/setup_ovs.py

```diff
@@ -271,15 +271,16 @@ def _handle_ovs_setup(pub_ip, ql, cluster, ext_etcd='', branch='', config_mgmt='
                 remote_con.process.execute('echo "deb http://apt.openvstorage.org {0} main" > /etc/apt/sources.list.d/ovsaptrepo.list'.format(ql))
                 remote_con.process.execute('apt-get update')
                 remote_con.process.execute('apt-get install -y ntp')
-                remote_con.process.execute('cd /tmp; wget http://10.100.129.100:8080/view/volumedriver/view/ubuntu/job/volumedriver-dev-release-ubuntu-16.04/lastSuccessfulBuild/artifact/volumedriver-core/build/debian/volumedriver-base_6.0.3-dev.201609011619.db51fbb_amd64.deb')
-                remote_con.process.execute('cd /tmp; wget http://10.100.129.100:8080/view/volumedriver/view/ubuntu/job/volumedriver-dev-release-ubuntu-16.04/lastSuccessfulBuild/artifact/volumedriver-core/build/debian/volumedriver-server_6.0.3-dev.201609011619.db51fbb_amd64.deb')
-                remote_con.process.execute('cd /tmp; wget http://10.100.129.100:8080/view/alba2/job/alba_docker_generic_package_ubuntu-16.04/lastSuccessfulBuild/artifact/alba_0.9.19_amd64.deb')
-                remote_con.process.execute('cd /tmp; wget http://10.100.129.100:8080/view/alba2/job/arakoon_docker_generic_package_ubuntu-16.04/lastSuccessfulBuild/artifact/arakoon_1.9.11_amd64.deb')
+                remote_con.process.execute('cd /tmp; wget http://10.100.129.100:8080/view/volumedriver/view/all/job/volumedriver-no-dedup-dev-release-ubuntu-16.04/10/artifact/volumedriver-core/build/debian/volumedriver-no-dedup-base_6.2.0-dev.201609302025.1fbbe1d_amd64.deb')
+                remote_con.process.execute('cd /tmp; wget http://10.100.129.100:8080/view/volumedriver/view/all/job/volumedriver-no-dedup-dev-release-ubuntu-16.04/10/artifact/volumedriver-core/build/debian/volumedriver-no-dedup-server_6.2.0-dev.201609302025.1fbbe1d_amd64.deb')
+                remote_con.process.execute('cd /tmp; wget http://10.100.129.100:8080/view/alba2/job/alba_docker_generic_package_ubuntu-16.04/lastSuccessfulBuild/artifact/alba_0.9.21_amd64.deb')
+                remote_con.process.execute('cd /tmp; wget http://10.100.129.100:8080/view/alba2/job/arakoon_docker_generic_package_ubuntu-16.04/lastSuccessfulBuild/artifact/arakoon_1.9.13_amd64.deb')
```

Comment:
I would recommend using a build from the alba/arakoon github releases page: see https://github.com/openvstorage/arakoon/releases and https://github.com/openvstorage/alba/releases .
Currently these might be exactly the same packages (haven't checked), but what's available at the jenkins link might change when we build new package - which you probably don't want?